src/main/java/bio/terra/service/snapshot/SnapshotTable.java

```diff
@@ -38,6 +39,13 @@ public List<Column> getColumns() {
     return columns;
   }
 
+  /**
+   * @return Columns formatted for use with Azure Synapse
+   */
+  public List<SynapseColumn> getSynapseColumns() {
+    return columns.stream().map(Column::toSynapseColumn).toList();
+  }
```

Comment:
This is the same as the code in `DatasetTable`. Could it be moved to the interface `Table` to avoid the code duplication?

Reply:
Oh good point - I'll move.